visidata/save.py

```diff
@@ -109,6 +109,8 @@ def saveSheets(vd, givenpath, *vsheets, confirm_overwrite=True):
     if not vsheets: # blank tuple
         vd.warning('no sheets to save')
         return
+    unloaded = [ vs for vs in vsheets if vs.rows is UNLOADED ]
+    vd.sync(*vd.ensureLoaded(unloaded))
```

Comment:
Closer!  But ensureLoaded is on BaseSheet, not the vd object.  And it also checks for UNLOADED itself.  So maybe we should make `vd.sync` ignore None values, and just call `vd.sync(*[vs.ensureLoaded() for vs in vsheets])`.